.pre-commit-config.yaml

```diff
@@ -49,3 +49,5 @@ repos:
         additional_dependencies:
           - types-protobuf
           - types-pyyaml
+          - types-requests
+          - google-api-python-client
```

Comment:
what is pulling in these types?

Reply:
I'm not sure. I'd accidentally removed more lines than I meant to from the exclude list in mypy.ini, so it might have been something in there that I forgot to double check after I put them back. In which case, we may need them again in the future, but for this PR, removing this line doesn't cause an issue.

Reply:
Correction, `pre-commit` runs fine locally, but in the action here it errors with
```
prefab_pb2.pyi:7: error: Cannot find implementation or library stub for module named "google"  [import]
```
I'll try to figure out why this error isn't popping up locally for me.